lib/ansible/module_utils/cloudstack.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+#
+# (c) 2015, René Moser <mail@renemoser.net>
+#
+# This code is part of Ansible, but is an independent component.
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+
+try:
+    from cs import CloudStack, CloudStackException, read_config
+except ImportError:
+    print("failed=True " + \
+        "msg='python library cs required: pip install cs'")
+    sys.exit(1)
```

Comment:
as in modules, we do not wan't to print/exit, capture the missing library in a variable and report it missing in a fail_json call.